### PR TITLE
Feature/cc service integration

### PR DESCRIPTION
**Pull request by hrytskivr**
This topic will provide more insight on how to create your own Control Center service.

### CODE REVIEW

File: controlcenter/service_integration.rst [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+   In here you will define your metric sources which you have to
+   reference later on in **GraphConfigs**.
+
+   Important things are **ID** field and **Metrics** list. Each metric
+   in **Metrics** list will have **IDs** as well and these must start
+   with **ID** of a **metric config** it's sitting in followed by a dot.
+
+   Take a look here => `MetricConfigs
+   example <https://gist.github.com/hrytskivr/ff0cb66bb4c230f0e391d43e5f38ccaf>`__.
```
Comment (byshovets) [inline controlcenter/service_integration.rst]: Your personal Gist account doesn't look like appropriate storage

File: controlcenter/service_integration.rst [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+   -  beware that you'll probably face
+      ``ControlCenterError: registry image collision`` when updating
+      your image at development stage and sadly there is no fix for this
+      yet
+
+-  .. rubric:: Command
+      :name: command
+
+   This is what **Serviced** will treat as a service itself and run when
+   you start your CC service. You can think of it as an **ENTRYPOINT**
```
Comment (byshovets) [inline controlcenter/service_integration.rst]: Would be nice to have a link to `ENTRYPOINT` documentation on Docker site

File: controlcenter/service_integration.rst [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+^^^^^
+
+Custom service in CC should have it's own container.
+
+In order to avoid problems, such as important env variables missing, you
+should use **zenoss/centos-base** as a base image in your
+**Dockerfile**.
+
+When you're done building your container you should tag & push it to
+**Docker Hub**, you should write a **Makefile** which willl take care of
```
Comment (byshovets) [inline controlcenter/service_integration.rst]: It is possible to use your own Docker registry, it is not limited to Docker Hub

**Comment by pcarinhas** [comment]
There is no integration point for this doc to be visible in SphinxDoc. You need to add a line in 
controlcenter/index.rst so that this gets build and has an index link.

**Comment by hrytskivr** [comment]
All of the requested changes has been made.
@byshovets @pcarinhas please take a look, thanks.

**Pull request merged**